### YamlMime:ManagedReference
items:
- uid: System.Windows.Data.ValueUnavailableException
  id: ValueUnavailableException
  children:
  - System.Windows.Data.ValueUnavailableException.#ctor
  - System.Windows.Data.ValueUnavailableException.#ctor(System.String)
  - System.Windows.Data.ValueUnavailableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Windows.Data.ValueUnavailableException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ValueUnavailableException
  nameWithType: ValueUnavailableException
  fullName: System.Windows.Data.ValueUnavailableException
  type: Class
  summary: "Wyjątek zgłaszany przez &lt;xref:System.Windows.Data.BindingGroup.GetValue%28System.Object%2CSystem.String%29?displayProperty=fullName&gt; metody, gdy wartość nie jest dostępna."
  syntax:
    content: 'public class ValueUnavailableException : SystemException'
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Windows.Data.ValueUnavailableException.#ctor
  id: '#ctor'
  parent: System.Windows.Data.ValueUnavailableException
  langs:
  - csharp
  name: ValueUnavailableException()
  nameWithType: ValueUnavailableException.ValueUnavailableException()
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException()
  type: Constructor
  assemblies:
  - PresentationFramework
  namespace: System.Windows.Data
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Windows.Data.ValueUnavailableException&quot;> </xref> klasy."
  remarks: "Ten konstruktor inicjuje <xref:System.Exception.Message%2A>Właściwość nowego wystąpienia na komunikat dostarczany przez system, który opisuje błąd, takie jak &quot;Wystąpił błąd systemowy&quot;.</xref:System.Exception.Message%2A> Ten komunikat bierze pod uwagę bieżące ustawienia kulturowe systemu.       W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Windows.Data.ValueUnavailableException>.</xref:System.Windows.Data.ValueUnavailableException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Odwołanie o wartości null (`Nothing` w języku Visual Basic). |   | <xref:System.Exception.Message%2A>| Ciąg z komunikatem o zlokalizowane. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ValueUnavailableException ();
    parameters: []
  overload: System.Windows.Data.ValueUnavailableException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Windows.Data.ValueUnavailableException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Windows.Data.ValueUnavailableException
  langs:
  - csharp
  name: ValueUnavailableException(String)
  nameWithType: ValueUnavailableException.ValueUnavailableException(String)
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException(String)
  type: Constructor
  assemblies:
  - PresentationFramework
  namespace: System.Windows.Data
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Windows.Data.ValueUnavailableException&quot;> </xref> klasy z określonym komunikatem."
  remarks: "Zawartość `message` parametr powinien być zrozumiała dla użytkownika. Element wywołujący tego konstruktora jest wymagana, aby upewnić się, że ten ciąg został zlokalizowany dla bieżącego ustawienia kulturowego systemu.       W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Windows.Data.ValueUnavailableException>.</xref:System.Windows.Data.ValueUnavailableException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Odwołanie o wartości null (`Nothing` w języku Visual Basic). |   | <xref:System.Exception.Message%2A>| Ciąg z komunikatem o. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ValueUnavailableException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Komunikat opisujący błąd."
  overload: System.Windows.Data.ValueUnavailableException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Windows.Data.ValueUnavailableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Windows.Data.ValueUnavailableException
  langs:
  - csharp
  name: ValueUnavailableException(SerializationInfo,StreamingContext)
  nameWithType: ValueUnavailableException.ValueUnavailableException(SerializationInfo,StreamingContext)
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - PresentationFramework
  namespace: System.Windows.Data
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Windows.Data.ValueUnavailableException&quot;> </xref> z zserializowanymi danymi."
  syntax:
    content: protected ValueUnavailableException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Obiekt zawierający dane zserializowanego obiektu."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Informacje kontekstowe dotyczące źródła lub miejsca docelowego."
  overload: System.Windows.Data.ValueUnavailableException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Windows.Data.ValueUnavailableException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Windows.Data.ValueUnavailableException
  langs:
  - csharp
  name: ValueUnavailableException(String,Exception)
  nameWithType: ValueUnavailableException.ValueUnavailableException(String,Exception)
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException(String,Exception)
  type: Constructor
  assemblies:
  - PresentationFramework
  namespace: System.Windows.Data
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Windows.Data.ValueUnavailableException&quot;> </xref> z określonego komunikatu o błędzie i odwołania do wyjątek wewnętrzny będący przyczyną tego wyjątku."
  remarks: "Wyjątek zgłaszany bezpośrednio w wyniku poprzedniego wyjątku może zawierać odwołania do poprzedniego wyjątku we <xref:System.Exception.InnerException%2A>Właściwości.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>Właściwość zwraca tę samą wartość, która została przekazana do konstruktora lub odwołanie o wartości null (`Nothing` w języku Visual Basic) Jeśli <xref:System.Exception.InnerException%2A>Właściwości nie dostarcza do konstruktora wartości wyjątku wewnętrznego.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.SystemException>.</xref:System.SystemException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Wyjątek wewnętrzny odwołania. |   | <xref:System.Exception.Message%2A>| Ciąg z komunikatem o. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ValueUnavailableException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Komunikat o błędzie opisujący przyczynę wyjątek."
    - id: innerException
      type: System.Exception
      description: "Wyjątek będący przyczyną bieżącego wyjątku. Jeśli `innerException` parametr nie jest odwołanie o wartości null (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> w języku Visual Basic), bieżący wyjątek jest zgłaszany w <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> bloku, który obsługuje wyjątek wewnętrzny."
  overload: System.Windows.Data.ValueUnavailableException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Windows.Data.ValueUnavailableException.#ctor
  parent: System.Windows.Data.ValueUnavailableException
  isExternal: false
  name: ValueUnavailableException()
  nameWithType: ValueUnavailableException.ValueUnavailableException()
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException()
- uid: System.Windows.Data.ValueUnavailableException.#ctor(System.String)
  parent: System.Windows.Data.ValueUnavailableException
  isExternal: false
  name: ValueUnavailableException(String)
  nameWithType: ValueUnavailableException.ValueUnavailableException(String)
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Windows.Data.ValueUnavailableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Windows.Data.ValueUnavailableException
  isExternal: false
  name: ValueUnavailableException(SerializationInfo,StreamingContext)
  nameWithType: ValueUnavailableException.ValueUnavailableException(SerializationInfo,StreamingContext)
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Windows.Data.ValueUnavailableException.#ctor(System.String,System.Exception)
  parent: System.Windows.Data.ValueUnavailableException
  isExternal: false
  name: ValueUnavailableException(String,Exception)
  nameWithType: ValueUnavailableException.ValueUnavailableException(String,Exception)
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Windows.Data.ValueUnavailableException.#ctor*
  parent: System.Windows.Data.ValueUnavailableException
  isExternal: false
  name: ValueUnavailableException
  nameWithType: ValueUnavailableException.ValueUnavailableException
